fuzz/fuzz_targets/helpers.rs

```diff
@@ -51,3 +57,19 @@ fn create_location(font: &FontRef, axis_positions: &[f32]) -> Location {
         .collect::<Vec<_>>();
     font.axes().location(raw_location)
 }
+
+/// Makes fuzzing agnostic of collection and non-collection inputs
+pub(crate) fn fonts(data: &[u8]) -> impl Iterator<Item = FontRef<'_>> {
+    let fr = FileRef::new(data);
+    if let Ok(FileRef::Collection(cr)) = fr {
+        let _ = cr.len();
+        let _ = cr.is_empty();
+    }
+    match FileRef::new(data) {
+        Ok(fr) => Some(fr.fonts()),
+        Err(_) => None,
+    }
+    .into_iter()
+    .flatten()
+    .filter_map(|i| i.ok())
```

Comment:
Is the `Iterator` of fonts a bit much and creating too many similar iterations, slowing down the fuzzer? How about letting the fuzzer randomness choose the ttc index? That would allow the fuzzing to cover the ttc over time, but focus on the interesting paths, instead of repeating iterations on the same safe/non-interesting data.

Reply:
Good point, made it so.